packages/dashboard-frontend/src/containers/__tests__/FactoryLoader.spec.tsx

```diff
@@ -667,6 +696,34 @@ describe('Factory Loader container', () => {
   });
 
   describe('Use a devfile V2', () => {
+    it('should resolve the factory with default policy and open existing workspace', async () => {
+      const location = 'http://test-location?new';
```

Comment:
As I see it, the `new` query param means nothing here, right? If so, can it be removed to avoid misleading?

Reply:
I've amended a new commit